client/components/external-link/index.jsx

```diff
@@ -41,6 +42,10 @@ export default React.createClass( {
 			rel: 'external'
 		} );
 
+		if ( props.target ) {
```

Comment:
If you specify an arbitrary target like `banana`, you still seem to get a `window.opener` in the resulting new window. I thought it was best to be safe and always append the `rel`.